keyboards/nibell/micropad4x4/readme.md

```diff
@@ -0,0 +1,15 @@
+# Micropad 4x4
+
+![Micropad 4x4](https://imgur.com/ugfJLXJ.jpg)
+
+A budget-friendly 4x4 Macropad
+
+* Keyboard Maintainer: [nibell](https://github.com/Nibell)
+Hardware Supported: Raspberry Pi Pico 
+* Hardware Availability: [GitHub](https://github.com/Nibell/Keyboards/tree/main/micropad4x4)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make nibell/micropad4x4:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
No way to enter the bootloader?  
Otherwise, bootloader section should be re-added

Reply:
Added bootloaderkey in layer and update docs